react-datetime.d.ts

```diff
@@ -50,17 +50,17 @@ declare module ReactDatetime {
      only parameter, if the date in the input is valid. If it isn't, the value
      of the input (a string) is returned.
      */
-    onChange?:(x: string) => void;
+    onChange?:(x: string|any) => void;
     /*
      Callback trigger for when the user opens the datepicker.
      */
-    onFocus?: (e : FocusEvent) => void;
+    onFocus?: () => void;
     /*
      Callback trigger for when the user clicks outside of the input, simulating a regular onBlur.
      The callback receives the selected moment object as only parameter, if the date in the
      input is valid. If it isn't, the value of the input (a string) is returned.
      */
-    onBlurs?: (e : FocusEvent) => void;
+    onBlur?: (e : string|any) => void;
```

Comment:
Same thing about `onBlur` as I mentioned above with `onChange`. I'm a bit confused, I'd love an explanation if you have one :)